File: Source/DroolTool.Models/DataTransferObjects/NewsAndAnnouncements/NewsAndAnnouncementsDto.cs
using System;
using System.Collections.Generic;
using System.Text;
using DroolTool.Models.DataTransferObjects.User;

namespace DroolTool.Models.DataTransferObjects
{
    public class NewsAndAnnouncementsDto
    {
        public int NewsAndAnnouncementsID { get; set; }
        public DateTime Date { get; set; }
        public string Title { get; set; }
        public string Link { get; set; }
        public string FileResourceGUIDAsString { get; set; }
        public UserSimpleDto LastUpdatedByUser { get; set; }
        public DateTime LastUpdatedDate { get; set; }

    }
}

File: Source/DroolTool.Web/src/app/shared/models/news-and-announcements/news-and-announcements-dto.ts
import { UserSimpleDto } from '../user/user-simple-dto';

export class NewsAndAnnouncementsDto{

    public NewsAndAnnouncementsID: number;
    public Title:string;
    public Link?: string;
    public Date: Date;
    public LastUpdatedByUser:UserSimpleDto;
    public LastUpdatedDate: Date;
    public FileResourceGUIDAsString: string;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}
